barretenberg/cpp/src/barretenberg/honk/sumcheck/sumcheck_output.hpp

```diff
@@ -15,31 +16,7 @@ template <typename Flavor> struct SumcheckOutput {
     std::vector<FF> challenge_point;
     // Evaluations in `u` of the polynomials used in Sumcheck
     ClaimedEvaluations purported_evaluations;
-
-    SumcheckOutput()
```

Comment:
There's no reason to define a bunch of constructors/operators that implement default behavior. Rather than update these to include the new member `verified`, I deleted them to get the same behavior.